plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/navigator/internal/N4JSResourceLinkHelper.java

```diff
@@ -54,9 +52,13 @@
 	@Inject
 	private ExternalLibraryWorkspace externalLibraryWorkspace;
 
+	@Inject
+	private LanguageSpecificURIEditorOpener languageSpecificURIEditorOpener;
+
 	@Inject
 	private N4JSProjectExplorerHelper helper;
 
+	// XXX: Obsolete when virtual nodes are removed
```

Comment:
Remove this comment?

Reply:
Thanks, I changed it into a todo for #1259